.travis.yml

```diff
@@ -1,6 +1,3 @@
 language: node_js
 node_js:
-  - "0.10"
-branches:
-  only:
-  - master
+  - "0.12"
```

Comment:
The `branches` configuration tells Travis which branches it should be concerned with. We only want Travis builds to run when we are opening a PR against a branch we care about. Previously, that was `master`, but now, we are working on the `v1.0` branch. Instead of removing the branch restriction, change the branch to `v1.0`.